src/dmd/doc.d

```diff
@@ -1844,6 +1856,53 @@ extern (C++) const(char)* skipwhitespace(const(char)* p)
     return p;
 }
 
+/************************************************
+ * Scan past the given characters.
+ */
```

Comment:
Please use Ddoc style comments.

Reply:
It is a Ddoc comment, just no sections.

Reply:
I was attempting to match the comment style used by the rest of doc.d. I'd be happy to change it.

Reply:
It's missing `Params:` and `Returns:` sections.